sql/testdata/select

```diff
@@ -120,3 +120,27 @@ SELECT *, "from", kw."from" FROM kw
 ----
 from from from
 1    1    1
+
+# SELECT from index.
+
+statement ok
+CREATE TABLE xyz (
+  x INT PRIMARY KEY,
+  y INT,
+  z INT,
+  CONSTRAINT foo INDEX (z, y)
+)
+
+statement ok
+INSERT INTO xyz VALUES (4, 5, 6), (1, 2, 3);
+
+query II
+SELECT * from xyz@foo
+----
+3 2
+6 5
+
+query I
+SELECT z from test.xyz@foo WHERE y = 5
```

Comment:
can you add `SELECT x FROM test.xyz@foo` to exercise the error case? also nit `from` should be capitalized